Mathmagician.Tests/UserInterfaceTests.cs

```diff
@@ -0,0 +1,126 @@
+
+            //assert
+            Assert.IsFalse(myUserInterfaceLowerCaseCommand.CheckUserCommandInput("fibonacci"));
+        }
+
+        //Checks to see if the user command (in mixed case) is a valid command, (it should return False, in that the command is NOT invalid)
+        [TestMethod]
+        public void UserInterface_TheUserMixedCaseCommandIsValid()
+        {
+            //arrange
+            UserInterface myUserInterfaceMixedCaseCommand = new UserInterface();
+
+            //assert
+            Assert.IsFalse(myUserInterfaceMixedCaseCommand.CheckUserCommandInput("fiBoNAcci".ToLower()));
+        }
+
+        //Checks to see if the user "number of values" is a valid number, (it should return True, in that the command is invalid)
+        [TestMethod]
+        public void UserInterface_TheUserNumberOfValuesIsValid()
+        {
+            //arrange
+            UserInterface myUserInterface = new UserInterface();
+
+            //assert
+            Assert.IsTrue(myUserInterface.CheckUserNumberInput("ten"));
+        }
+
+        //Checks to see if the user "number of values" is a valid number, (it should return False, in that the command is NOT invalid)
+        [TestMethod]
+        public void UserInterface_TheUserNumberOfValuesIsInvalid()
+        {
+            //arrange
+            UserInterface myUserInterface = new UserInterface();
+
+            //assert
+            Assert.IsFalse(myUserInterface.CheckUserNumberInput("10"));
+        }
+
+        //The InitiateNumberOperation calls the individual operation method and assigns the returned results to the List returnedOperationResult.
+        // The default of the switch statement should never be called but assigns a single element to returnedOperationResult = 96500
+        [TestMethod]
+        public void UserInterface_CheckTheExcecutionOfTheInitiateNumberOperation()
+        {
+            //arrange
+            UserInterface myUserInterface = new UserInterface();
+
+            //act
+            myUserInterface.InitiateNumberOperation("", 0);
+
+            //assert
+            Assert.AreEqual(new List<int> { 96500 }[0], myUserInterface.returnedOperationResult[0]);
+        }
+
+        //Sets the returnedOperationResult List to {2, 66, 78 } and then calls the ReturnedOperationValues method which put the list values in a string
+        // This checks to see if the returned thing is a String and also the correct string
+        [TestMethod]
+        public void UserInterface_IsTheResultsStringInFactTheCorrectString()
+        {
+            //arrange
+            UserInterface myUserInterface = new UserInterface();
+            myUserInterface.returnedOperationResult = new List<int> { 2, 66, 78 };
+
+            //assert
+            Assert.AreEqual("String", myUserInterface.ReturnedOperationValues().GetType().Name);
+            Assert.AreEqual("2 66 78 ", myUserInterface.ReturnedOperationValues());
+        }
+    }
```

Comment:
You can add a couple unit tests that use CollectionAssert.AreEqual to determine if all the items in the lists you are comparing are equal.